cleverhans/evaluation.py

```diff
@@ -100,6 +100,40 @@ def correctness_and_confidence(sess, model, x, y, batch_size=None,
 
   return out
 
+def run_attack(sess, model, x, y, attack, attack_params, batch_size=None,
+               devices=None, feed=None):
+  """
+  Run attack on every example in a dataset.
+  :param sess: tf.Session
+  :param model: cleverhans.model.Model
+  :param x: numpy array containing input examples (e.g. MNIST().x_test )
```

Comment:
See comments above on naming x and y arguments

Reply:
These match the other functions in the file